certbot/cli.py

```diff
@@ -1156,9 +1156,10 @@ def _create_subparsers(helpful):
                 " Currently --csr only works with the 'certonly' subcommand.")
     helpful.add("revoke",
                 "--reason", dest="reason",
-                choices=CaseInsensitiveList(constants.REVOCATION_REASONS.keys()),
+                choices=CaseInsensitiveList(sorted(constants.REVOCATION_REASONS,
```

Comment:
because I wanted them to be in order according to their numerical value (as specified in the rfc)
It's not mega important, but I think it's nice